channels/sig_analog.c

```diff
@@ -1766,6 +1766,7 @@ static void *__analog_ss_thread(void *data)
 
 	if (!chan) {
 		/* What happened to the channel? */
+		ast_debug(1, "Huh? No Asterisk channel provided for DAHDI channel %d?\n", p->channel);
```

Comment:
If `chan` being `NULL` here is a programming error then this should probably be an `ast_assert()`. If it’s a runtime error then it’s fine as an `ast_debug()`.